client/src/includes/sidePanel.js

```diff
@@ -2,6 +2,7 @@ import { ngettext } from '../utils/gettext';
 
 export default function initSidePanel() {
   const sidePanelWrapper = document.querySelector('[data-form-side]');
+  const content = document.querySelector('.content');
```

Comment:
```suggestion
```

Let's just set this globally, it's simpler and also avoids us grabbing some random DOM element to attach to. Normally we avoid JS selecting something by css classes also.